Resources/Locale/en-US/deltav/job/job-description.ftl

```diff
@@ -1,2 +1,2 @@
 job-description-medical-borg = Half-human, Half-machine. Follow your laws, serve the crew, and assist the medical department.
-job-description-courier = Deliver mail and other packages from and to logistics. Avoid dogs.
```

Comment:
Courier beware... your description isn't there

Reply:
Oops, committing that now.